packages/sdk-react/README.md

```diff
@@ -1 +1,223 @@
 # @turnkey/sdk-react
+
+The `@turnkey/sdk-react` package simplifies the integration of the Turnkey API into React-based applications. It builds on top of the `@turnkey/sdk-browser` package, enabling developers to implement authentication and wallet functionalities using React components.
+
+## Overview
+
+- **Authentication**: Supports email, passkey, phone, and social logins.
+- **Wallet Operations**: Import and export wallets securely.
+- **Client Utilities**: Includes `passkeyClient`, `authIframeClient`, and more.
+- **Components**: Abstracts auth, session, import and export logic away from the developer and allows for simple easy to use imports
+- **Customization**: Components theming options to align with your application's design.
```

Comment:
```
**Customization**: Components theming options to align with your application's design.
```

--> `Theming options for your components to align with your application's design`